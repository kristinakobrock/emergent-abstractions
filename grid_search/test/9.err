  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:02, 19.08it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 23.34it/s] 20%|##################4                                                                         | 10/50 [00:00<00:01, 27.92it/s] 28%|#########################7                                                                  | 14/50 [00:00<00:01, 28.45it/s] 34%|###############################2                                                            | 17/50 [00:00<00:01, 25.85it/s] 40%|####################################8                                                       | 20/50 [00:00<00:01, 24.93it/s] 46%|##########################################3                                                 | 23/50 [00:00<00:01, 23.97it/s] 52%|###############################################8                                            | 26/50 [00:01<00:00, 24.31it/s] 58%|#####################################################3                                      | 29/50 [00:01<00:00, 22.59it/s] 64%|##########################################################8                                 | 32/50 [00:01<00:00, 24.35it/s] 70%|################################################################3                           | 35/50 [00:01<00:00, 25.68it/s] 78%|#######################################################################7                    | 39/50 [00:01<00:00, 28.02it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 27.24it/s] 90%|##################################################################################8         | 45/50 [00:01<00:00, 25.40it/s] 96%|########################################################################################3   | 48/50 [00:01<00:00, 26.02it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.14it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 25.79it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 23.78it/s] 77%|######################################################################7                     | 10/13 [00:00<00:00, 28.34it/s]100%|############################################################################################| 13/13 [00:00<00:00, 25.85it/s]100%|############################################################################################| 13/13 [00:00<00:00, 25.98it/s]
