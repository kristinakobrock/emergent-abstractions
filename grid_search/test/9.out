# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=64", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6667352914810181, "acc": 0.6211743950843811, "length": 3.527724504470825, "mode": "train", "epoch": 1}
{"loss": 0.6754488348960876, "acc": 0.6171875, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.663594126701355, "acc": 0.6254253387451172, "length": 3.528388023376465, "mode": "train", "epoch": 2}
{"loss": 0.6680088639259338, "acc": 0.6296326518058777, "length": 1.1171875, "mode": "test", "epoch": 2}
{"loss": 0.6624861359596252, "acc": 0.6246827244758606, "length": 3.518004894256592, "mode": "train", "epoch": 3}
{"loss": 0.6485441327095032, "acc": 0.6345458030700684, "length": 3.56640625, "mode": "test", "epoch": 3}
{"loss": 0.6605144143104553, "acc": 0.6256603598594666, "length": 3.5206332206726074, "mode": "train", "epoch": 4}
{"loss": 0.645739734172821, "acc": 0.635540783405304, "length": 3.953125, "mode": "test", "epoch": 4}
{"loss": 0.6514710187911987, "acc": 0.62868332862854, "length": 3.481083631515503, "mode": "train", "epoch": 5}
{"loss": 0.6383507251739502, "acc": 0.6460937857627869, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.6317987442016602, "acc": 0.6529694199562073, "length": 3.6519038677215576, "mode": "train", "epoch": 6}
{"loss": 0.62985759973526, "acc": 0.6494140028953552, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.6042253375053406, "acc": 0.6746844053268433, "length": 3.8060505390167236, "mode": "train", "epoch": 7}
{"loss": 0.6377413272857666, "acc": 0.6578125357627869, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.5890575051307678, "acc": 0.6865150332450867, "length": 3.7988736629486084, "mode": "train", "epoch": 8}
{"loss": 0.6372716426849365, "acc": 0.6564452648162842, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.571475088596344, "acc": 0.7022467255592346, "length": 3.864621162414551, "mode": "train", "epoch": 9}
{"loss": 0.6161203384399414, "acc": 0.6876954436302185, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.5505641102790833, "acc": 0.7251098155975342, "length": 3.8975656032562256, "mode": "train", "epoch": 10}
{"loss": 0.6010395288467407, "acc": 0.7093749642372131, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.5322454571723938, "acc": 0.7345665097236633, "length": 3.9335994720458984, "mode": "train", "epoch": 11}
{"loss": 0.5994744300842285, "acc": 0.7039062976837158, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.5217359662055969, "acc": 0.7414259314537048, "length": 3.9473302364349365, "mode": "train", "epoch": 12}
{"loss": 0.5964931845664978, "acc": 0.7001953721046448, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.509568989276886, "acc": 0.7518555521965027, "length": 3.9506142139434814, "mode": "train", "epoch": 13}
{"loss": 0.5733188986778259, "acc": 0.724609375, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.4960341155529022, "acc": 0.7625986337661743, "length": 3.9627974033355713, "mode": "train", "epoch": 14}
{"loss": 0.5712941288948059, "acc": 0.7166014909744263, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.48375558853149414, "acc": 0.77101731300354, "length": 3.966660499572754, "mode": "train", "epoch": 15}
{"loss": 0.5554619431495667, "acc": 0.7220702767372131, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.4715690612792969, "acc": 0.7786160707473755, "length": 3.974128246307373, "mode": "train", "epoch": 16}
{"loss": 0.5568535923957825, "acc": 0.7302734851837158, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.4570791721343994, "acc": 0.7880183458328247, "length": 3.9812755584716797, "mode": "train", "epoch": 17}
{"loss": 0.5556298494338989, "acc": 0.7361328601837158, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.44825005531311035, "acc": 0.7934711575508118, "length": 3.982628107070923, "mode": "train", "epoch": 18}
{"loss": 0.5458783507347107, "acc": 0.7404296398162842, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.43867558240890503, "acc": 0.7983672618865967, "length": 3.985788106918335, "mode": "train", "epoch": 19}
{"loss": 0.5465397834777832, "acc": 0.7501952648162842, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.42848294973373413, "acc": 0.8044706583023071, "length": 3.988513946533203, "mode": "train", "epoch": 20}
{"loss": 0.5469154119491577, "acc": 0.7570312023162842, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.4289659857749939, "acc": 0.8075310587882996, "length": 3.989461660385132, "mode": "train", "epoch": 21}
{"loss": 0.5308973789215088, "acc": 0.7544922232627869, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.4216538965702057, "acc": 0.8091710209846497, "length": 3.9898765087127686, "mode": "train", "epoch": 22}
{"loss": 0.5280671119689941, "acc": 0.7527344226837158, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.41747593879699707, "acc": 0.81255704164505, "length": 3.988953113555908, "mode": "train", "epoch": 23}
{"loss": 0.5294851660728455, "acc": 0.7562499642372131, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.4131966233253479, "acc": 0.8140629529953003, "length": 3.992107391357422, "mode": "train", "epoch": 24}
{"loss": 0.516567051410675, "acc": 0.7595702409744263, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.4080994725227356, "acc": 0.8175214529037476, "length": 3.993034839630127, "mode": "train", "epoch": 25}
{"loss": 0.5179103016853333, "acc": 0.765625, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.3986953794956207, "acc": 0.8223623037338257, "length": 3.992687702178955, "mode": "train", "epoch": 26}
{"loss": 0.5077511072158813, "acc": 0.7638672590255737, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.40075400471687317, "acc": 0.8204632997512817, "length": 3.9944374561309814, "mode": "train", "epoch": 27}
{"loss": 0.5075677037239075, "acc": 0.7691406607627869, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.3975215256214142, "acc": 0.8209443688392639, "length": 3.995121717453003, "mode": "train", "epoch": 28}
{"loss": 0.5080198645591736, "acc": 0.767578125, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.39080163836479187, "acc": 0.8269222378730774, "length": 3.995893716812134, "mode": "train", "epoch": 29}
{"loss": 0.5043018460273743, "acc": 0.7644531726837158, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.3844941258430481, "acc": 0.8317756056785583, "length": 3.9957988262176514, "mode": "train", "epoch": 30}
{"loss": 0.4997076988220215, "acc": 0.7681640982627869, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.3882458806037903, "acc": 0.8306126594543457, "length": 3.9961941242218018, "mode": "train", "epoch": 31}
{"loss": 0.49741989374160767, "acc": 0.7708984017372131, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.37972450256347656, "acc": 0.8360027074813843, "length": 3.9964094161987305, "mode": "train", "epoch": 32}
{"loss": 0.4938318133354187, "acc": 0.7707031965255737, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.3773181438446045, "acc": 0.8382303714752197, "length": 3.9971444606781006, "mode": "train", "epoch": 33}
{"loss": 0.49650269746780396, "acc": 0.7716797590255737, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.3798869550228119, "acc": 0.8393791317939758, "length": 3.996302366256714, "mode": "train", "epoch": 34}
{"loss": 0.4945862889289856, "acc": 0.7685546278953552, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.3730480372905731, "acc": 0.8401680588722229, "length": 3.99715256690979, "mode": "train", "epoch": 35}
{"loss": 0.5063655972480774, "acc": 0.7708984017372131, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.3724447786808014, "acc": 0.8429819941520691, "length": 3.9975788593292236, "mode": "train", "epoch": 36}
{"loss": 0.4979919195175171, "acc": 0.7708984017372131, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.37070149183273315, "acc": 0.8433207273483276, "length": 3.997124433517456, "mode": "train", "epoch": 37}
{"loss": 0.48645681142807007, "acc": 0.7761719226837158, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.3700031638145447, "acc": 0.8445129990577698, "length": 3.998152732849121, "mode": "train", "epoch": 38}
{"loss": 0.4843219518661499, "acc": 0.7861328721046448, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.3661036789417267, "acc": 0.8469148874282837, "length": 3.9981906414031982, "mode": "train", "epoch": 39}
{"loss": 0.4875619411468506, "acc": 0.7874999642372131, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.35896965861320496, "acc": 0.8475586175918579, "length": 3.9981048107147217, "mode": "train", "epoch": 40}
{"loss": 0.48524010181427, "acc": 0.7865234017372131, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.3584151864051819, "acc": 0.8497117757797241, "length": 3.997286319732666, "mode": "train", "epoch": 41}
{"loss": 0.4931822419166565, "acc": 0.7804687023162842, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.35529741644859314, "acc": 0.8507294654846191, "length": 3.9985461235046387, "mode": "train", "epoch": 42}
{"loss": 0.48101431131362915, "acc": 0.7931640148162842, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.35090717673301697, "acc": 0.8538298010826111, "length": 3.998457670211792, "mode": "train", "epoch": 43}
{"loss": 0.47628456354141235, "acc": 0.7943359017372131, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.3498707413673401, "acc": 0.8534272909164429, "length": 3.9987664222717285, "mode": "train", "epoch": 44}
{"loss": 0.48337018489837646, "acc": 0.7898436784744263, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.34889763593673706, "acc": 0.8557695150375366, "length": 3.9989540576934814, "mode": "train", "epoch": 45}
{"loss": 0.480887234210968, "acc": 0.7898436784744263, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.35006284713745117, "acc": 0.8544849753379822, "length": 3.999007225036621, "mode": "train", "epoch": 46}
{"loss": 0.4835328459739685, "acc": 0.7994140982627869, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.34198588132858276, "acc": 0.8577611446380615, "length": 3.9990410804748535, "mode": "train", "epoch": 47}
{"loss": 0.487304151058197, "acc": 0.7914062142372131, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.3385801315307617, "acc": 0.8605656623840332, "length": 3.9987995624542236, "mode": "train", "epoch": 48}
{"loss": 0.4969322085380554, "acc": 0.7802734971046448, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.3407883644104004, "acc": 0.8588191270828247, "length": 3.9991824626922607, "mode": "train", "epoch": 49}
{"loss": 0.49859607219696045, "acc": 0.7796874642372131, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.34022992849349976, "acc": 0.8581686615943909, "length": 3.9989829063415527, "mode": "train", "epoch": 50}
{"loss": 0.49179691076278687, "acc": 0.78515625, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.33947426080703735, "acc": 0.8601074814796448, "length": 3.9991443157196045, "mode": "train", "epoch": 51}
{"loss": 0.47591280937194824, "acc": 0.7939453125, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.3318006098270416, "acc": 0.8657296895980835, "length": 3.9992992877960205, "mode": "train", "epoch": 52}
{"loss": 0.47957247495651245, "acc": 0.8003906607627869, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.3304472267627716, "acc": 0.8639756441116333, "length": 3.9994006156921387, "mode": "train", "epoch": 53}
{"loss": 0.47913891077041626, "acc": 0.7974609732627869, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.3247118592262268, "acc": 0.866714358329773, "length": 3.999155282974243, "mode": "train", "epoch": 54}
{"loss": 0.48291462659835815, "acc": 0.7894530892372131, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.3273743987083435, "acc": 0.867545485496521, "length": 3.999361276626587, "mode": "train", "epoch": 55}
{"loss": 0.4753984212875366, "acc": 0.7929686903953552, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.3244709372520447, "acc": 0.8694854378700256, "length": 3.9996163845062256, "mode": "train", "epoch": 56}
{"loss": 0.48964589834213257, "acc": 0.7917968034744263, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.3245095908641815, "acc": 0.8684896230697632, "length": 3.9995508193969727, "mode": "train", "epoch": 57}
{"loss": 0.47605830430984497, "acc": 0.7992186546325684, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.3218310475349426, "acc": 0.869426429271698, "length": 3.999631881713867, "mode": "train", "epoch": 58}
{"loss": 0.4737010598182678, "acc": 0.8033203482627869, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.3153705298900604, "acc": 0.8734065294265747, "length": 3.9995219707489014, "mode": "train", "epoch": 59}
{"loss": 0.48648005723953247, "acc": 0.7943359017372131, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.31249192357063293, "acc": 0.8732215762138367, "length": 3.9989616870880127, "mode": "train", "epoch": 60}
{"loss": 0.47581350803375244, "acc": 0.8009766340255737, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7281666398048401
